vault/resource_pki_secret_backend_role.go

```diff
@@ -286,11 +286,38 @@ func pkiSecretBackendRoleResource() *schema.Resource {
 				Type:        schema.TypeList,
 				Required:    false,
 				Optional:    true,
-				Description: "Specify the list of allowed policies IODs.",
+				Description: "Specify the list of allowed policies OIDs.",
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
 			},
+			"policy_identifier": {
```

Comment:
Would adding `ConflictsWith` to this schema field help in this situation? If we specify that this new field conflicts with the legacy `policy_identifiers` field, then TF would log an error to the user if they try to use both fields within the same config. ([here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_jwt_auth_backend.go#L59) is an example of using `ConflictsWith`)

I see we currently log a warning to the console and ignore the legacy field; maybe explicitly erroring might be better?

Reply:
Awesome, I did not know about this.